src/main/java/tech/sourced/siva/SivaCLI.java

```diff
@@ -0,0 +1,30 @@
+package tech.sourced.siva;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class SivaCLI {
+
+    public static void main(String args[]) {
```

Comment:
Not sure what you mean here.
I would prefer keeping CLI, as it's already been proven useful for testing and finding bugs.